recipes/lzstring/meta.yaml

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
GitHub is being weird about which line this comment is for (overview says line 1, detail view says line 4). Do you mean at the top of the file (just removed in the latest commit) or between the `{% set sha256` line and `package:`?